### ISSUE TITLE
Missing keyboard navigation mechanism

### ISSUE THREAD

**Issue by meganvilla** [OpenSpace/OpenSpace#3104]
There is no mechanism currently to allow users to move through the main sections of  the application. Users must tab through the interface, which at the moment is  prevented due to the inaccessibility of many controls. But such an approach is  inefficient and time consuming for users that rely on keyboard navigation. 

**Suggested solution:** 
Provide common shortcuts, e.g. F6 to circle focus between main sections of the  application while allowing users to tab or use Arrow keys within a group of controls.

**Comment by alexanderbock** [comment]
From my view, keyboard support is now fully possible with the exception of closing individual tabs.  @engbergandreas feel free to and please reopen the issue if you disagree.
